### PR TITLE
user_agent added and closing session after  activity pull API call. 

### PR DESCRIPTION
**Pull request by sagargg**
- Ref [documentation](https://gitlab.com/datopian/clients/data-subscriptions/-/blob/master/data_subscriptions/worker/latest_ckan_activity.py#L19-22)  user_agent is missing so, added. 
-  Closing the session after activity pull. This implementation is done Ref. https://github.com/ckan/ckanapi#session-control

### CODE REVIEW

File: data_subscriptions/worker/latest_ckan_activity.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def __init__(self, url=None, start_time=None, proposed_limit=1_000):
         self.current_offset = 0
 
     def __call__(self):
-        api = RemoteCKAN(self.url)
+        api = RemoteCKAN(self.url, user_agent='ckanapiexample/1.0 (+http://example.com/my/website)')
```
Comment (anuveyatsu) [inline data_subscriptions/worker/latest_ckan_activity.py:19]: @sagargg why not to use actual domain?

File: data_subscriptions/worker/latest_ckan_activity.py [hunk 2]
```diff
@@ -28,6 +28,7 @@ def __call__(self):
             self.current_offset += self.server_limit
             if self.has_reached_the_end(response):
                 break
+            RemoteCKAN.close(api)
```
Comment (anuveyatsu) [inline data_subscriptions/worker/latest_ckan_activity.py:31]: @sagargg can we implement using 

```
with RemoteCKAN(self.url, user_agent=...) as api:
    ...
```

File: data_subscriptions/worker/latest_ckan_activity.py (hunk 1 above)
Comment (sagargg) [reply data_subscriptions/worker/latest_ckan_activity.py:19]: @anuveyatsu  not sure but found mostly user-age in this pattern. 
`Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.97 Safari/537.36`

Comment (anuveyatsu) [reply data_subscriptions/worker/latest_ckan_activity.py:19]: @sagargg I'm not sure if it is needed at all, can we we just not do it?

File: data_subscriptions/worker/latest_ckan_activity.py (hunk 2 above)
Comment (sagargg) [reply data_subscriptions/worker/latest_ckan_activity.py:31]: @anuveyatsu  I tried but tests are failing. 

Screen Shot 2020-08-14 at 12 23 25

Comment (anuveyatsu) [reply data_subscriptions/worker/latest_ckan_activity.py:31]: @sagargg ok, let's remove user agent then and merge this.

Comment (sagargg) [reply data_subscriptions/worker/latest_ckan_activity.py:31]: @anuveyatsu  removed.

**Pull request merged**